lib/fastlane/plugin/bundletool/actions/bundletool_action.rb

```diff
@@ -158,30 +158,37 @@ def self.puts_error!(message)
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :ks_path,
+                                       env_name: 'FL_BUNDLETOOL_STORE_FILE',
```

Comment:
Is it ok if we rename it to `FL_BUNDLETOOL_KEYSTORE_FILE`?

Reply:
Sure! Changed the variable name to `FL_BUNDLETOOL_KEYSTORE_FILE` as suggested